man/man5/zpool-features.5

```diff
@@ -27,9 +27,9 @@ ZFS pool on\-disk format versions are specified via "features" which replace
 the old on\-disk format numbers (the last supported on\-disk format number is
 28). To enable a feature on a pool use the \fBupgrade\fR subcommand of the
 zpool(8) command, or set the \fBfeature@\fR\fIfeature_name\fR property
-to \fBenabled\fR. Please also note the operation of the \fBcompatibility\fR
-property in zpoolprops(8) which allows sets of features to be specified
-together.
+to \fBenabled\fR. Please also see the
+.Sx "Compatibility feature sets"
```

Comment:
This doesn't work; the Sx macro is not being interpreted correctly here. Would appreciate help from anyone who understands mdoc better than I.